src/ast/passes/semantic_analyser.cpp

```diff
@@ -2731,10 +2743,30 @@ void SemanticAnalyser::visit(AssignMapStatement &assignment)
   Visit(assignment.map);
   Visit(assignment.expr);
 
+  auto type = assignment.expr->type;
+
+  if (!IsValidAssignment(type, assignment.expr->is_map)) {
+    auto hint = "";
+    if (type.IsHistTy()) {
+      hint = "hist(retval)";
+    } else if (type.IsLhistTy()) {
+      hint = "lhist(rand %10, 0, 10, 1)";
```

Comment:
`rand %10`?

Reply:
It's the same example for lhist that we have in the adoc